server/eclipse-project/src/main/java/us/freeandfair/corla/json/CountyDashboardRefreshResponse.java

```diff
@@ -267,7 +269,7 @@ protected CountyDashboardRefreshResponse(final Long the_id,
     my_audited_ballot_count = the_audited_ballot_count;
     my_discrepancy_count = the_discrepancy_count;
     my_disagreement_count = the_disagreement_count;
-    my_ballot_under_audit_id = the_ballot_under_audit_id;
+    my_ballot_under_audit_ids = the_ballot_under_audit_ids;
```

Comment:
are we supposed to know which ballot under audit belongs to which audit board?

Reply:
Yeah, if you look at how this is generated, it's a list of size `number_of_audit_boards` where each index in the list maps to an audit board, and there's a CVR ID for that audit board to look at next or `null` if the board has nothing left to do.

Reply:
It's up to the client to look at the right audit board index to get its next CVR (as well as looking itself up in the audit board signed-in data structure), that's the last card in the epic and what Sara and I are working on today.